lib/main_common.pm

```diff
@@ -612,6 +612,7 @@ sub load_extra_tests {
         loadtest "console/zypper_moo";
         loadtest "console/gpg";
         loadtest "console/shells";
+        loadtest "console/dstat";
```

Comment:
Yes, that's better. But I don't know if that test would work on SLE. There is a section in L581 for "opensuse-only". At best you can conduct a test that it also works for current SLE in development, i.e. SLE15, if not then just move this line to L581 and following.

Reply:
I put it there to have it run on SLE and openSUSE. dstat is in SLE too.

Reply:
Please forgive me being a bit late to answer (in holiday). If dstat also works in SLE that is the best solution but we should make sure the package is available where we expect it and also test that including the creation of the SLE specific needles that are necessary. Please see https://wiki.microfocus.net/index.php?title=openQA#Use_the_fetchneedles_script_for_sles_.28recommended.29 as a hint how to get SLE needles if not already done so. I found out that dstat for SLE15 is in the module "server applications" so it should be fine to call that on any non-minimal SLES15 installation so our extra_tests_in_textmode, e.g. https://openqa.suse.de/tests/latest?arch=x86_64&flavor=Installer-DVD&distri=sle&test=extra_tests_in_textmode&version=15&machine=64bit, should be fine. Could you test SLE15 and also at least one maintenance tests, e.g. clone the one mentioned above and https://openqa.suse.de/tests/latest?distri=sle&flavor=Server-DVD-Updates&arch=x86_64&version=12-SP3&machine=64bit&test=mau-extratests ? Otherwise we can also ask a helpful SUSE QA or QAM engineer to test on SLE

Reply:
sorry, totally lost track of this. can we go the 'helpful QA engineer' route? ;)

Reply:
And now *I* lost track :D Yes, we will test it. I created a ticket for our team in https://progress.opensuse.org/issues/32719